trunk/SUAVE/Methods/Aerodynamics/Airfoil_Panel_Method/veldis.py

```diff
@@ -0,0 +1,72 @@
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+# veldis.py 
+#
+# Created:  Mar 2021, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units
+import numpy as np
+
+# ----------------------------------------------------------------------
+# veldis
+# ----------------------------------------------------------------------
+
+## @ingroup Methods-Aerodynamics-Airfoil_Panel_Method
+def veldis(qg,x,y,xbar,ybar,st,ct,al,npanel):
+    """Compute the tangential velocity distribution at the       
+                 midpoint of each panel   
+
+    Assumptions:
+    None  
+    
+    Inputs:                                                    
+
+     qg          -  Vector of source/np.sink and vortex strengths              
```

Comment:
line these up

also np.sink?

Reply:
I did a control "F" to change something and this was accidentally changed. fixed